Dreary/README.md

```diff
@@ -0,0 +1,35 @@
+# Drery
+
+## Screenshots
+
+### BIB
+![BIB Screenshot](bib.png)
+
+### Deep
+![Deep Screenshot](deep.png)
+
+### Psycho
+![Psycho Screenshot](psycho.png)
+
+### Deeper
+![Deeper Screenshot](deeper.png)
+
+### Lover
+![Lover Screenshot](lover.png)
+
+### RainforestCafe
+![Rainforest Cafe Screenshot](rainforestcafe.png)
+
+##Important
+Sidebar Config option must be enabled for theme to work properly, you can use it without, however some features may not be available.
```

Comment:
What are you referring to?

Reply:
@morpheusthewhite I'm referring to the sidebar changes, such as the border around playlists, I can see about making it work without but I figured that for now its fine.